docs/about/overview/radio-settings.mdx

```diff
@@ -1,67 +1,97 @@
 ---
 id: radio-settings
 title: Radio Settings
+slug: /overview/radio-settings
 sidebar_label: Radio Settings
-sidebar_position: 3
+sidebar_position: 1
 ---
 
-We use the same channel maps as LoRaWAN (though this is not LoRaWAN).
-(Note, not the same channel map as TTN for US frequencies.)
+:::info
+Meshtastic is **not** LoRaWAN, nor TTN (TheThingsNetwork). However, Meshtastic does use the same channel maps as LoRaWAN.
+:::
```

Comment:
We are not using the LoraWan bandplan either. We use the full spectrum at the chosen spacing, which can mean several hundred possible channels/frequencies in the US region.